doc/python/map-configuration.md

```diff
@@ -20,27 +20,32 @@ jupyter:
     name: python
     nbconvert_exporter: python
     pygments_lexer: ipython3
-    version: 3.10.4
+    version: 3.10.0
   plotly:
-    description: How to configure and style base maps for Choropleths and Bubble Maps.
+    description: How to configure and style base maps for outline-based Geo Maps.
     display_as: maps
     language: python
     layout: base
-    name: Map Configuration and Styling
+    name: Map Configuration and Styling on Geo Maps
     order: 12
     page_type: u-guide
     permalink: python/map-configuration/
     thumbnail: thumbnail/county-level-choropleth.jpg
 ---
 
-### Mapbox Maps vs Geo Maps
+### Tile Maps vs Outline Maps
 
 Plotly supports two different kinds of maps:
 
-1. **Mapbox maps** are [tile-based maps](https://en.wikipedia.org/wiki/Tiled_web_map). If your figure is created with a `px.scatter_mapbox`, `px.line_mapbox`, `px.choropleth_mapbox` or `px.density_mapbox` function or otherwise contains one or more traces of type `go.Scattermapbox`, `go.Choroplethmapbox` or `go.Densitymapbox`, the `layout.mapbox` object in your figure contains configuration information for the map itself.
-2. **Geo maps** are outline-based maps. If your figure is created with a `px.scatter_geo`, `px.line_geo` or `px.choropleth` function or otherwise contains one or more traces of type `go.Scattergeo` or `go.Choropleth`, the `layout.geo` object in your figure contains configuration information for the map itself.
+- **[Tile-based maps](https://en.wikipedia.org/wiki/Tiled_web_map)**
 
-This page documents Geo outline-based maps, and the [Mapbox Layers documentation](/python/mapbox-layers/) describes how to configure Mapbox tile-based maps.
+If your figure is created with a `px.scatter_map`, `px_scatter_mapbox`, `px.line_map`, `px.line_mapbox`, `px.choropleth_map`, `px.choropleth_mapbox`, `px.density_map`, or `px.density_mapbox` function or otherwise contains one or more traces of type `go.Scattermap`, `go.Scattermapbox`,`go.Choroplethmap`, `go.Choroplethmapbox`, `go.Densitymap`, or `go.Densitymapbox` the `layout.map` object in your figure contains configuration information for the map itself.
```

Comment:
```suggestion
If your figure is created with a `px.scatter_map`, `px.scatter_mapbox`, `px.line_map`, `px.line_mapbox`, `px.choropleth_map`, `px.choropleth_mapbox`, `px.density_map`, or `px.density_mapbox` function or otherwise contains one or more traces of type `go.Scattermap`, `go.Scattermapbox`,`go.Choroplethmap`, `go.Choroplethmapbox`, `go.Densitymap`, or `go.Densitymapbox` the `layout.map` object in your figure contains configuration information for the map itself.
```